db/db_impl/db_impl_write.cc

```diff
@@ -1801,7 +1804,24 @@ Status DBImpl::SwitchMemtable(ColumnFamilyData* cfd, WriteContext* context) {
   }
   if (s.ok()) {
     SequenceNumber seq = versions_->LastSequence();
-    new_mem = cfd->ConstructNewMemtable(mutable_cf_options, seq);
+    std::unique_ptr<uint32_t> new_filterbits;
```

Comment:
This seems a little unusual. Why not just use an automatic uint32_t with a value of 0 indicating no auto-tuning?

Reply:
That sounds like a much better idea - I'm making the change right now